backend/src/api/routes/userOrganizationRoute.ts

```diff
@@ -1,17 +1,13 @@
 import { Router, Request } from 'express';
 import { run } from '../../common/helpers/routeHelper';
-import {
-  getUsers,
-  updateUserOrganization,
-  createUserOrganization,
-  resendInvite } from '../../services/userOrganizationService';
+import * as UOSerevice from '../../services/userOrganizationService';
 
 const router = Router();
 
 router
-  .get('/:organizationId', run((req: Request) => getUsers(req.params.organizationId)))
-  .post('/', run((req: Request) => createUserOrganization(req.body)))
-  .put('/', run((req: Request) => updateUserOrganization(req.body)))
-  .post('/resend', run((req: Request) => resendInvite(req.body)));
+  .get('/:organizationId', run((req: Request) => UOSerevice.getUsers(req.params.organizationId)))
```

Comment:
no need to set explicitly `req: Request` because the `req`'s type has been already set implicitly